src/sass/components/_shop-section-shop.scss

```diff
@@ -0,0 +1,75 @@
+.section--shop {
+    .shop {
+        background: $page-header-back;
+        height: 340px; 
+
+        &-heading {
+            padding: 35px 0;
+            color: $page-header-font;
+            position: relative;
+
+            &-h1 {
+                text-transform: uppercase;
+                margin: 0;
+                font-size: 24px;
+            }
+        }
+    }
+}
+
+.breadcrumb {
+    margin: 0;
+    padding: 0;
+    background: none;
+    margin-right: 40px;
+
+    &-item.active {
```

Comment:
tutaj bym się zagnieździł bo potem i tak robisz działania na `item` więc spróbuj to połączyć